kitsune/wiki/tests/test_parser.py

```diff
@@ -339,30 +334,23 @@ def test_button_image_for_nesting(self):
 
     def test_direct_recursion(self):
         """Make sure direct recursion is caught on the very first nesting."""
-        d = document(title='Template:Boo')
-        d.save()
+        d = TemplateDocumentFactory(title='Template:Boo')
 
         # Twice so the second revision sees content identical to itself:
-        for i in range(2):
-            revision(document=d, content='Fine [[Template:Boo]] Fellows',
-                     is_approved=True).save()
+        for _ in range(2):
+            ApprovedRevisionFactory(document=d, content='Fine [[Template:Boo]] Fellows')
```

Comment:
I think you can do this with `.build_batch()` if you wanted to.

https://factoryboy.readthedocs.org/en/latest/reference.html#factory.build_batch